flask-server/main.py

```diff
@@ -377,16 +377,14 @@ def fetch_write_clinks():
 @app.route('/apis/fetch-bookmarks', methods=['GET'])
 def fetch_bookmarks():
     resp_token = decode_auth_token(request.headers.get('Authorization'))
-
+    clink_title = request.headers.get('title')
```

Comment:
We should really use the clink's id to fetch its bookmarks. We can get the clink's title with the id. This api shall actually be `/apis/fetch-bookmarks/:id` such that you can get the id from the param instead of putting it in the title. The header in an HTTP request should only contain metadata.

Reply:
made a change, not entirely sure if it what you wanted